test/language/statements/async-function/syntax-declaration-no-line-terminator.js

```diff
@@ -0,0 +1,13 @@
+// Copyright 2016 Microsoft, Inc. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+author: Brian Terlson <brian.terlson@microsoft.com>
+es7id: pending
+description: Async function declarations cannot have a line break after `async`
+info: Reference error is thrown due to looking up async in strict mode
+flags: [onlyStrict]
+negative: ReferenceError
```

Comment:
Since this is a runtime error, could you assert this with `assert.throws`? I still have hopes for gh-196